### PR TITLE
Update interactive docs

### PR DESCRIPTION
**Pull request by mrocklin**
This updates our interactive/Jupyter documentation.  We focus a bit more now on JupyterLab, and remove the link to the pangeo docs, which was a bit stale.

### CODE REVIEW

File: docs/source/interactive.rst [hunk 1]
```diff
@@ -5,99 +5,138 @@ Dask-jobqueue is most often used for interactive processing using tools like
...
+SSH Tunneling
+-------------
+
+If your personal machine is on the same network as your cluster, then you can
+ignore this step.
+
+If you are on a different network (like your home network), and have to SSH in,
+then it can be difficult to have your local web browser connect to the Jupyter
+server running on the HPC machine.  If your institution doesn't have something
+like `JupyterHub <https://jupyter.org/hub>`_ set up, then the easiest way to
```
Comment (guillaumeeb) [inline docs/source/interactive.rst]: Why not talking about Jupyterhub from the start? The same sentence or nearly in introduction, to say that you don't want to do any of this if a Jupyterhub is available (except if launched notebook are too old, or to be able to use dask-labextension if not installed, but that's some advanced business).

File: docs/source/interactive.rst [hunk 2]
```diff
@@ -5,99 +5,138 @@ Dask-jobqueue is most often used for interactive processing using tools like
...
-           frameborder="0" allow="autoplay; encrypted-media"
-           allowfullscreen></iframe>
+      username@hostname$ jupyter lab --no-browser --ip="*" --port 8888
 
+    You might also run ``echo $HOSTNAME`` on that machine as well to see the
+    host name.
 
-Dask Dashboard with Jupyter
----------------------------
+-   ``hpc.agency.gov`` is the address that you usually use to
```
Comment (guillaumeeb) [inline docs/source/interactive.rst]: is it?

File: docs/source/interactive.rst [hunk 3]
```diff
@@ -5,99 +5,138 @@ Dask-jobqueue is most often used for interactive processing using tools like
...
-``nbserverproxy`` in the Python environment you use for launching the notebook,
-and activate it as indicated in the docs:
+So in a real example this might look like the following:
 
 .. code-block:: bash
 
-   pip install nbserverproxy
-   jupyter serverextension enable --py nbserverproxy
+    alice@login2.summit $ jupyter lab --no-browser --ip="login2" --port 8888
+    alice@laptop        $ ssh -L 8888:login2:8888 summit.olcf.ornl.gov
```
Comment (guillaumeeb) [inline docs/source/interactive.rst]: You didn't put `alice@summit`, is it intended?

File: docs/source/interactive.rst [hunk 4]
```diff
@@ -5,99 +5,138 @@ Dask-jobqueue is most often used for interactive processing using tools like
...
+
+Typically, the dashboard is served on a separate port from Jupyter, and so can
+be used whether you choose to use Jupyter or not.  If you want to open up a
+connection to see the dashboard you can do so with SSH Tunneling as described
+above.  The dashboard's default port is at ``8787``, and is configurable with
+the ``dashboard_address=`` keyword to the Dask Jobqueue cluster objects.
+
+However, Jupyter is also able to proxy the dashboard connection through the
+Jupyter server, allowing you to access the dashboard at
+``http://localhost:8888/proxy/8787/status``.  This requires no additional SSH
```
Comment (guillaumeeb) [inline docs/source/interactive.rst]: You don't need to install `nbserverproxy` or `jupyter-server-proxy` anymore ? It is automatically bundled with jupyter?

Comment (mrocklin) [reply docs/source/interactive.rst]: It is bundled with dask-labextension

File: docs/source/interactive.rst (hunk 3 above)
Comment (mrocklin) [reply docs/source/interactive.rst]: Thanks.  Fixed.

File: docs/source/interactive.rst (hunk 2 above)
Comment (mrocklin) [reply docs/source/interactive.rst]: I've made it more clear that these are placeholders.

File: docs/source/interactive.rst (hunk 1 above)
Comment (mrocklin) [reply docs/source/interactive.rst]: I've added a JupyterHub advertisement at the top.

**Comment by mrocklin** [comment]
Thanks for the feedback @guillaumeeb .  I think I've responded to all of your suggestions.

**Comment by guillaumeeb** [comment]
Thanks @mrocklin !

**Comment by mrocklin** [comment]
Thanks for the review!

**Pull request merged**